test/Recipe-test.js

```diff
@@ -91,6 +91,6 @@ describe('Recipe', function() {
   it('should be able calculate the cost of its ingredients', function() {
     const total = recipe.getTotalCost();
 
-    expect(total).to.equal(14836);
+    expect(total).to.equal(148.33);
```

Comment:
This is good -- it means the decimal is going in the right place now. But why is it $148.33 instead of $148.36? Where did those 3 cents go? Rounding differences somewhere? I don't think we need to worry about it at all, it just seems weird. It must be something to do with when things are getting rounded.

Reply:
math is HARD